FrontEnd/src/components/HeaderFooter/header/navbar/Logout.jsx

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../../../hooks";
+
+function Logout(props) {
```

Comment:
We do not use any props in the component it could be removed

Reply:
Done